### PR TITLE
Add open-loadflow voltageTargetPriorities parameter documentation

### PR DESCRIPTION
**Pull request by p-arvy**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

Documentation added for voltageTargetPriorities powsybl-open-loadflow documentation.
See https://github.com/powsybl/powsybl-open-loadflow/pull/955

**What is the current behavior?**

There is no such documentation.

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: pages/documentation/simulation/powerflow/openlf.md [hunk 1]
```diff
@@ -531,6 +531,14 @@ containing the generator terminals used as angle reference in the load flow calc
 There is one Terminal created/added in the extension for each calculated Synchronous Component.
 Works only when `referenceBusSelectionMode` is set to `GENERATOR_REFERENCE_PRIORITY`.
 
+**voltageTargetPriorities**  
+This parameter allows to configure the priority order between voltage targets. 
+By default, the order is `["GENERATOR", "TRANSFORMER", "SHUNT"]`.
```
Comment (jeandemanged) [inline pages/documentation/simulation/powerflow/openlf.md:536]: should add a note that GENERATOR includes indistinctively generators, static var compensators, and VSC HVDC converters.

**Review by jeandemanged — CHANGES_REQUESTED** [review]

**Review by Hadrien-Godard — APPROVED** [review]

File: pages/documentation/simulation/powerflow/openlf.md [hunk 2]
```diff
@@ -531,6 +531,17 @@ containing the generator terminals used as angle reference in the load flow calc
 There is one Terminal created/added in the extension for each calculated Synchronous Component.
 Works only when `referenceBusSelectionMode` is set to `GENERATOR_REFERENCE_PRIORITY`.
 
+**voltageTargetPriorities**  
+When multiple equipment regulate the same bus with different voltage targets,
+this parameter enables configuring priority to resolve inconsistencies by aligning the voltage targets.
+Priority is determined by equipment type order; the voltage target of the equipment type listed first takes precedence over those listed later. 
+By default, the order is `["GENERATOR", "TRANSFORMER", "SHUNT"]`.  
+Note that `"GENERATOR"` indistinctively includes generators, static var compensators, and VSC HVDC converters.
```
Comment (annetill) [inline pages/documentation/simulation/powerflow/openlf.md:539]: And batteries?

Comment (jeandemanged) [reply pages/documentation/simulation/powerflow/openlf.md:539]: If I am not mistaken today batteries in iIDM are not able to perform voltage control.

Comment (annetill) [reply pages/documentation/simulation/powerflow/openlf.md:539]: https://github.com/powsybl/powsybl-open-loadflow/pull/1004

**Review by annetill — APPROVED** [review]

**Review by jeandemanged — APPROVED** [review]

**Pull request merged**